### ISSUE TITLE
About the localization methods

### ISSUE THREAD

**Issue by FFFox-abc** [rohban-lab/Knowledge_Distillation_AD#18]
When I test the localization method, the RocAUC is different every time. the results are fluctuate within 0.5 roughly.
my config is:

# Data parameters
experiment_name: 'localization_local_equal_net'
dataset_name: mvtec # [mnist, fashionmnist, cifar10, mvtec, retina]
last_checkpoint: 600

# Training parameters
num_epochs: 601 # mnist/fashionmnist:51, cifar10:201, mvtec:601
batch_size: 32
learning_rate: 1e-3
mvtec_img_size: 128

normal_class: 'toothbrush' # mvtec:'capsule', mnist:3        

lamda: 0.5 # mvtec:0.5, Others:0.01

pretrain: True # True:use pre-trained vgg as source network --- False:use random initialize
use_bias: False # True:using bias term in neural network layer
equal_network_size: False # True:using equal network size for cloner and source network --- False:smaller network for cloner
direction_loss_only: False
continue_train: True

# Test parameters
localization_test: True  # True:For Localization Test --- False:For Detection
localization_method: 'smooth_grad' # gradients , smooth_grad , gbp

**Comment by FFFox-abc** [comment]
fix random seed
